volttron/utils/rmq_setup.py

```diff
@@ -573,7 +568,7 @@ def _verify_and_save_instance_ca(rmq_config, instance_ca_path, instance_ca_key):
 
 
 def setup_rabbitmq_volttron(setup_type, verbose=False, prompt=False, instance_name=None,
-                            rmq_conf_file=None, env=None):
+                            rmq_conf_file=None, max_retries=12, env=None):
```

Comment:
Called in volttron/volttrontesting/fixtures/rmq_test_setup.py (line 189), but missing max_retries. Should it be added?

Reply:
Fixed in volttron/volttrontesting/fixtures/rmq_test_setup.py